srv/modules/utils/cluster.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=modernize-parse-error
+#
+# The salt-api calls functions with keywords that are not needed
+# pylint: disable=unused-argument
+"""
+Functions related to the ceph cluster
+"""
+
+
+from __future__ import absolute_import
+from __future__ import print_function
+import logging
+
+
+log = logging.getLogger(__name__)
+
+
+def name():
+    """
+    Return the cluster name from the ceph namespace, original namespace
+    or default to 'ceph'
+    """
+    if 'ceph' in __pillar__ and 'cluster' in __pillar__['ceph']:
+        return __pillar__['ceph']['cluster']
+    if 'cluster' in __pillar__:
+        return __pillar__['cluster']
+    return 'ceph'
+
+
+def help_():
+    """
+    Usage
+    """
+    usage = ('salt-run cluster.name: \n\n'
+             '    Returns a name of the cluster\n'
+             '\n\n')
+    print(usage)
+    return ""
+
```

Comment:
Does a utils module work as a pseudo runner?  That seems unexpected.

kbaikov and I were chatting about the name function in #1153.  I think the name function is fine when called from modules since the pillar setting implies membership.  However, for the subset of runners in Stage 0-2 should never use this since the admin would be setting the cluster based on command line argument or environment variable.  (This is for the fictional day in the future where we support multiple Ceph clusters with one Salt master.)

We never solved that issue and I don't know that we have to now.  I do think we need to have enough commentary to explain when the utils should not be used.